Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.9-alpine
 
 LABEL version="2.1"
```

```suggestion
LABEL version="2.7"
```

We'll cut a new release with this one, as it adds a feature

---

Sounds good, commited